dbt_platform_helper/domain/maintenance_page.py

```diff
@@ -55,21 +54,7 @@ def __str__(self):
         )
 
 
-def get_maintenance_page_type(session: boto3.Session, listener_arn: str) -> Union[str, None]:
-    lb_client = session.client("elbv2")
-
-    rules = lb_client.describe_rules(ListenerArn=listener_arn)["Rules"]
-    tag_descriptions = get_rules_tag_descriptions(rules, lb_client)
-
-    maintenance_page_type = None
-    for description in tag_descriptions:
-        tags = {t["Key"]: t["Value"] for t in description["Tags"]}
-        if tags.get("name") == "MaintenancePage":
-            maintenance_page_type = tags.get("type")
-
-    return maintenance_page_type
-
-
+# TODO should this be in its own provider, inside the VPC one, what logic is this sepcific too?
```

Comment:
Out of scope of load balancer provider but probably one which can be easily refactored if there is a case for it.